### ISSUE TITLE
Specifications sections for SVG docs claim to link to SVG1.1 but actually link to SVG2 (affects SVG <marker>: allowed values of orient attribute)

### ISSUE THREAD

**Issue by S-Rafael** [mdn/content#22061]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/orient

### What specific section or headline is this issue about?

Usage notes, Specifications

### What information was incorrect, unhelpful, or incomplete?

The link in the Specifications section claims to point to the SVG 1.1 Specification (Second Edition) but has the URL of the SVG 2 Draft specification.

### What did you expect to see?

If the cited specification is SVG 1.1 (Second Edition), the link should point there.

More broadly, there is a content problem with the description of the `orient` attribute. The current description is actually based on the SVG 2 draft specification despite overtly claiming to cite only SVG 1.1 specs.

The SVG 1.1 standard allows [only two types of values](https://www.w3.org/TR/SVG11/painting.html#OrientAttribute) in the `orient` attribute on a `<marker>` element: the literal `auto` or an angle.
The additional possible value of `auto-start-reverse` is only introduced in the [SVG 2 Draft specification](https://svgwg.org/svg2-draft/painting.html#OrientAttribute). It would be helpful to indicate this fact.

### Do you have any supporting links, references, or citations?

Orient attribute in SVG 1.1: https://www.w3.org/TR/SVG11/painting.html#OrientAttribute

Orient attribute in SVG 2: https://svgwg.org/svg2-draft/painting.html#OrientAttribute

**Comment by bsmth** [comment]
Hi @S-Rafael thanks a lot for raising the issue, we'll take a look into this one shortly!

**Comment by sideshowbarker** [comment]
I believe this happens because in https://github.com/w3c/browser-specs/blob/main/index.json, `https://svgwg.org/svg2-draft/` is listed at the `nightlyUrl` for both the `SVG11` spec and for the `SVG2` spec — and I guess our client code for consuming the browser-specs data takes the first match, which is for the `SVG11` spec.

I’m not sure how we should best handle this case. Heads-up to @Elchi3, @queengooborg, and @tidoust for ideas

**Comment by tidoust** [comment]
If the logic is to look at `series.nightlyUrl` and pick up the first match, I think it needs to be amended.

The nightly URL of a spec is under `nightly.url`. The [`series.nightlyUrl`](https://github.com/w3c/browser-specs/#seriesnightlyurl) property contains the "unversioned URL that allows to access the latest Editor's Draft of the **current specification in the series**".

The SVG 1.1 entry in browser-specs has `https://www.w3.org/TR/SVG11/` under `nightly.url`. Its `series.nightlyUrl` property is `https://svgwg.org/svg2-draft/` because the current specification is SVG2 and that's the best unversioned URL for the SVG series.

In other words, if you look at `series.nightlyUrl` to find the right entry in browser-specs, I think the logic should be completed to take the entry for which `entry.shortname === entry.series.currentSpecification`. This will give you the current specification in the series, SVG 2 in this case.

**Comment by queengooborg** [comment]
The current logic can be found here: https://github.com/mdn/yari/blob/f149f4b4101ade4e9ba3396a9368187edf378854/build/document-extractor.ts#L649-L655

Basically, we're checking _all_ spec URLs: `spec.url`, `nightly.url`, `nightly.alternateUrls` and `series.nightlyUrl`.  Until now, we hadn't run into an issue regarding conflicting spec versions, or at least that we know of.  I think that adding logic to ensure that the result from `nightlyUrl` is always the latest spec version is a great plan.  I'll submit a PR!

**Comment by S-Rafael** [comment]
Thanks @queengooborg for tackling this issue!

As for the secondary, content-related problem, should we split it off into a separate ticket?
The content part of the issue was brought to my attention by the Gitlab user [KrIr17](https://gitlab.com/KrIr17), in the context of the incorrect versioning of the `orient` attribute in Inkscape (open-source SVG editor). [Inkscape issue for reference](https://gitlab.com/inkscape/inkscape/-/issues/3935).

To re-iterate what content improvement I'd like to see: make it clear which values of `orient` are allowed by the SVG 1.1 standard and which are an SVG 2 extension.

**Comment by queengooborg** [comment]
I have to give the credit to @tidoust for this one, their insight and recommendation was the real work! 😃 

> As for the secondary, content-related problem, should we split it off into a separate ticket?

I didn't even realize there was a content problem as well!  Yes, I think creating a separate issue would be best!
